### DISCUSSION TITLE
Nomi dei dispositivi - dubbio prima di iniziare!

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2285] [GENERALE - Discussioni generiche]
Ciao a tutti, 
ho collegato tramite home kit molti dispositivi e ora come potete immaginare mi ha importato i nomi di default dei dispostivi come per esempio "Switch 22", "Light device 1", "Lightbulb" etc etc.
stavo normalizzando i nomi quando mi è sorto un dubbio!
dal momento che alcuni di questi dispositivi li utilizzerò anche con alexa, mi potete consigliare come gestire la nomenclatura?
mi spiego meglio... il mio dubbio è se devo inserire la parola "LUCE" oppure posso evitarla, fi faccio un esempio:
il dispositivo per esempio si chiama "Luce_Dispensa" l'entità l'ho rinominata "Luce Dispensa" (senza _) e ID entità in "light.luce_dispensa_lightbulb"
come vedete ho sempre riportato la parola LUCE che però a mio discapito la ritrovo quando creo la card (salvo che no modifico)
allo stesso tempo però il fatto di riportare LUCE èl'ho pensato in quanto con alexa 
"accendi luce dispensa" altriemnti sarebbe "accendi Dispensa"
voi cosa consigliate?

**Comment by 50494554524F** [comment]
per il nome di un dispositivo fisico puoi usare

xxx.marca_tipo_id_sensore

quindi per un termometro, ad esempio, ti verrà

sensor.xiaomi_thermometer_1234_temperature
sensor.xiaomi_thermometer_1234_humidity

in questo modo potrai crearci sopra un dispositivo logico che avrà le sensori/funzioni che ti interessano (potrai copiarle tutte, aggiungerne, toglierne) e nel caso cambiassi il dispositivo fisico dovrai solo cambiare i nomi nel dispositivo logico senza impazzire con tutto il resto...
per il dispositivo logico puoi usare nomi più pertinenti, del tipo

xxx.piano_stanzaoarea_dispositivo_sensore

quindi ti verrà qualcosa del tipo

sensor.1st_floor_living_thermometer_temperature
sensor.1st_floor_living_thermometer_humidity

analogamente per automazioni script etc....

per un pulsante / luci avrai

switch.1st_floor_living_lampolightobulb_qualcosaaltro

light.1st_floor_kitchen_dispensa

etc etc

questo per gli id, per i nomi metterai quello che vuoi mostrare nella dashboard
